docs/pages/appendices.md

```diff
@@ -303,6 +303,6 @@ unsupported.
 
 #### Oxford Nanopore
 
-* **PromethION** - as of Release 19.01.1
-* _MinION - as of Release 18.12_
+* **PromethION** - as of PromethION Release 19.01.1
+* _MinION - as of MinION Release 18.12_
```

Comment:
It looks like the "MinION" label is missing an underscore, but should it be double-asterisked instead to match the PromethION label?

Reply:
The MinION Processor isn't 100% done yet, it'll be bolded instead of italicized once I've verified it's done. I've italicized the whole line for PacBio Sequel and MinION so it's harder to miss